lib/npm.js

```diff
@@ -436,50 +440,24 @@ class Npm {
     return usage(this)
   }
 
-  // output to stdout in a progress bar compatible way
-  output (...msg) {
-    log.clearProgress()
-    // eslint-disable-next-line no-console
-    console.log(...msg.map(Display.clean))
-    log.showProgress()
+  forceLog (...args) {
+    this.#display.forceLog(...args)
```

Comment:
This is a new method because in the exit-handler we did a hack to swap out the loglevel so we always show the update notifier. So now you can call `forceLog` to have it always shown.

This would be another case for proc-log allowing arbitrary metadata to allow consumers to hint that logs should always be shown. This would still respect silent but not anything else.